dsm/resource_group.go

```diff
@@ -321,3 +323,25 @@ func isSetApprovalPolicy(d *schema.ResourceData, m interface{}) bool {
 	}
 	return false
 }
+// handle the key undo policy for both create and update
+func set_key_undo_policy(d *schema.ResourceData, obj map[string]interface{}) {
+    if key_undo_policy_window_time, ok := d.GetOk("key_undo_policy_window_time"); ok {
+        key_history_policy := make(map[string]interface{})
+        key_history_policy["undo_time_window"] = key_undo_policy_window_time.(int) * 24 * 60 * 60
```

Comment:
This seems weird, why not just expose the underlying seconds api to the consumer instead?

Reply:
Actually, I followed UI. In UI, it represents in days.
It will be easy for the user to configure in days rather than seconds.
 
Please let me know, whether it makes sense.

Reply:
I disagree. The frontend is a client, but that doesn't mean you need to replicate all of its choices. I think it would be better to natively expose the underlying API than forcing the loss of resolution.

Reply:
I modified it accordingly, now the input will take it in seconds format.

Kindly, please approve.